cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -6180,6 +6180,7 @@ display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
+# TODO - need to add missing fields for this search to props.conf for Nessus:Pro
```

Comment:
I think all fields are added. can we remove this

Reply:
This don't need in the DOC PR anyways.